ui/core/individual_sim_ui.ts

```diff
@@ -583,6 +584,20 @@ export abstract class IndividualSimUI<SpecType extends Spec> extends SimUI {
 		] as Array<StatOption<IconInputConfig<Player<any>, any>>>);
 		otherBuffOptions.forEach(iconInput => makeIconInput(buffsSection, iconInput));
 
+		const revitalizeBuffOptions = this.splitRelevantOptions([
+			{ item: IconInputs.RevitalizeRejuvination, stats: [Stat.StatAttackPower, Stat.StatSpellPower] },
```

Comment:
If you want someone to apply to all sims, you can leave the `stats` list empty.

Reply:
4b65f50e86602d9d83103bf815d75845ceaa7329